opentelemetry-sdk/src/opentelemetry/sdk/_logs/__init__.py

```diff
@@ -12,27 +12,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
+from opentelemetry._logs import LogRecord
 from opentelemetry.sdk._logs._internal import (
-    LogData,
-    LogDeprecatedInitWarning,
     LogDroppedAttributesWarning,
     Logger,
     LoggerProvider,
     LoggingHandler,
     LogLimits,
-    LogRecord,
     LogRecordProcessor,
+    ReadableLogRecord,
+    ReadWriteLogRecord,
 )
 
 __all__ = [
-    "LogData",
+    "LogDroppedAttributesWarning",
     "Logger",
     "LoggerProvider",
     "LoggingHandler",
     "LogLimits",
     "LogRecord",
     "LogRecordProcessor",
-    "LogDeprecatedInitWarning",
```

Comment:
@xrmx I removed a lot of the code from #4771 my understanding those were temporary changes to let people know about this change, LogDeprecatedInitWarning is completely removed here